test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
Perhaps this PR should wait on https://github.com/marionettejs/backbone.marionette/issues/2201 to help alleviate concern.